index.html

```diff
@@ -12,8 +12,90 @@
     <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
   <![endif]-->
 </head>
-
-<body>
-  <h1>My Resume</h1>
-</body>
+<div class="container">
+  <header>
+    <h1>My Resume</h1>
+    <nav>
+      <a href="http://127.0.0.1:5500/#">Home</a>
+      <a href="#">About</a>
+      <a href="#">Work</a>
+      <a href="http://127.0.0.1:5500/less/Contact.html">Contact</a>
```

Comment:
Link doesn't work but also you linked it to the less folder directory. I'm sure it was by accident